### PR TITLE
Rework select.py, add number.py

### PR DESCRIPTION
**Pull request by bouwew**
Add regulation_mode selector, add max_boiler_temp (input) number

### CODE REVIEW

File: custom_components/plugwise/number.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    command: str
+
+
+@dataclass
+class PlugwiseNumberEntityDescription(
+    NumberEntityDescription, PlugwiseEntityDescriptionMixin
+):
+    """Class describing Plugwise Number entities."""
+
```
Comment (CoMPaTech) [inline custom_components/plugwise/number.py:34]: Why this empty class?

File: custom_components/plugwise/select.py [hunk 2]
```diff
@@ -1,23 +1,64 @@
...
+    command: str
+    current_option: str
+    options: str
+
+
+@dataclass
+class PlugwiseSelectEntityDescription(
+    SelectEntityDescription, PlugwiseSelectDescriptionMixin
+):
+    """Class describing Sensibo Number entities."""
```
Comment (CoMPaTech) [inline custom_components/plugwise/select.py:37]: Same comment + wrong stub naming (Sensibo)

File: custom_components/plugwise/number.py (hunk 1 above)
Comment (bouwew) [reply custom_components/plugwise/number.py:34]: Apparently this is how you create the top level class that recognizes the `command=` line.
This is brefa does it, and others in HA Core.

File: custom_components/plugwise/select.py (hunk 2 above)
Comment (bouwew) [reply custom_components/plugwise/select.py]: Sensibo naming replaced, thanks!

File: README.md [hunk 3]
```diff
@@ -28,14 +28,22 @@ Our [`python-plugwise`](https://github.com/plugwise/python-plugwise) python modu
...
+  - Rework select.py, add selector for the heating/cooling-system regulation mode.
+  - Add number.py (and tests), provide an option for changing the maximum boiler (water) temperature setpoint.
+  - Remove climate schedule attributes, moved to the options-attribute of the select schedule entity.
+  - Implement fix for the sticking HA persistent notifications
+
+## MAR 2022 [0.21.5]
  - Smile:
   - **BREAKING** improved naming of the locally present outdoor temp sensor connected to the OpenTherm device: `outdoor_air_temperature`. The former `sensor.opentherm_outdoor_temperature` is now visible as `sensor.opentherm_outdoor_air_temperature`. Use of the zipcode based `outdoor_temperature` has not changed.
-  - **POTENTIALLY BREAKING** for consistency renamed schema to schedule, i.e. the attributes will now be `available_schedules` and `selected_schedule`
+  - **POTENTIALLY BREAKING** for consistency renamed schema to schedule, i.e. the attributes will now be `available_schedules` and `selected_schedule`.
```
Comment (CoMPaTech) [inline README.md:42]: Do we really want to change them or obsolete them like suggested with Core? And/or we remove them from core here, right?

File: custom_components/plugwise/select.py [hunk 4]
```diff
@@ -43,18 +91,31 @@ def __init__(
...
 
     async def async_select_option(self, option: str) -> None:
-        """Change the selected option."""
-        await self.coordinator.api.set_schedule_state(
-            self.device.get("location"),
-            option,
-            SCHEDULE_ON,
-        )
+        """Change to the selected entity option."""
+        result = await self.async_send_api_call(option, self.entity_description.command)
```
Comment (CoMPaTech) [inline custom_components/plugwise/select.py:114]: I'd still recommend rewriting this to the approach we did in https://github.com/home-assistant/core/blob/dev/homeassistant/components/plugwise/select.py

**Review by CoMPaTech — APPROVED** [review]

File: custom_components/plugwise/select.py (hunk 4 above)
Comment (bouwew) [reply custom_components/plugwise/select.py:114]: I don't think that construct will work?

**Review by CoMPaTech — APPROVED** [review]

**Review by CoMPaTech — APPROVED** [review]

**Pull request merged**